package/main/daml/Daml.Finance.Claims/daml.yaml

```diff
@@ -4,21 +4,21 @@
 sdk-version: 2.5.0-snapshot.20221208.11077.0.53b7db71
 name: daml-finance-claims
 source: daml
-version: 0.1.6
+version: 0.1.7
```

Comment:
This and the below version bumps shouldn't be required I think. I did bump all versions over the weekend, and there wasn't any release since then. Or am I missing something?

Reply:
Im afraid I had a look at the wrong github tags, will revert the version cahnges